### ISSUE TITLE
Rebuild Native Modules Hangs (AGAIN) After VS Code Upgrade

### ISSUE THREAD

**Issue by scotdb** [ibmdb/vscode-extension#73]
Upgraded to VS Code 1.92.0.

Once again the rebuild of the native modules for Db2 Connect is hanging.

Have tried the usual "tricks" -

1) Reinstall Db2 Connect
2) Attempt the rebuild manually from the command palette.

Here's the output -

[8/7/2024 11:28:12 AM] Proceeding to rebuild native modules
Error: ENOENT: no such file or directory, access 'c:\Users\Philip Nelson\.vscode\extensions\ibm.db2connect-2.3.1\node_modules\ibm_db'
[8/7/2024 11:28:12 AM] No ibm_db found...
Downloading node ibm_db Driver from https://github.com/ibmdb/node-ibm_db/archive/master.zip ...
Downloading and extraction of node ibm_db Driver completed successfully.
Downloading took 9.7 seconds.
Successfully renamed the directory.
Using the existing CLI Driver
ODBC Driver path = C:\Program Files\IBM\SQLLIB

As always I use existing CLI driver since pulling one directly is not possible in our corporate environment.

This really needs to be addressed once and for all - we cannot afford to keep losing Db2 Connect every time we get a VS Code update and we can't afford to not install updates (security etc.)

**Comment by bimalkjha** [comment]
@scotdb Could you please retry now? Thanks.

**Comment by bimalkjha** [comment]
I have subscribed to the notification of new release from https://github.com/electron/electron and https://github.com/microsoft/vscode, so that I can get email whenever a new release happens. I started watch for both repo.
We have added support for electron 30 and 31 both in ibm_db driver. Latest version of vscode uses electron 30, so next few releases of vscode is also taken care as of now.
With timely notification, we'll be ahead of vscode release in providing support for new version of electron, so that users will not face such issue in future when vscode releases a new version. Thanks.
